src/caf/viz/__init__.py

```diff
@@ -1,3 +1,9 @@
 """CAF package containing functionality for transport related visualisations."""
 # TODO(MB) Write a more detailed description of the package
+
+from matplotlib import pyplot as _plt
+
 from ._version import __version__
+
+# Aliases
+from caf.viz.xy_plot import XYPlotType, axes_plot_xy, plot_xy
```

Comment:
Does this need to be here? I feel like it sets a precedent of accessing these functions this way, when really they should be accessed through their module?